### PR TITLE
Add DragonDrop

### PR DESCRIPTION
**Pull request by tresni**
Support for [DragonDrop](http://shinyplasticbag.com/dragondrop/) based on #3570

### CODE REVIEW

File: Casks/dragondrop.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+cask :v1 => 'dragondrop' do
+  version :latest
+  sha256 :no_check
+  
+  url 'http://shinyplasticbag.com/dragondrop/dragondrop.dmg'
+  name 'DragonDrop'
+  homepage 'http://shinyplasticbag.com/dragondrop/'
+  license :freemium
```
Comment (vitorgalvao) [inline Casks/dragondrop.rb]: Isn’t it a timed trial?

Comment (tresni) [inline Casks/dragondrop.rb]: Yeah good point. I can change to commercial since it does stop functioning after the trial ends.

Comment (vitorgalvao) [inline Casks/dragondrop.rb]: Please do.

**Comment by vitorgalvao** [comment]
There’s trailing whitespace that needs to be removed.

**Comment by tresni** [comment]
@vitorgalvao Latest commit passes travis CI tests and addresses issues you've noted.  Let me know if there is anything else you think needs to happen.

**Comment by vitorgalvao** [comment]
Just need you to please [squash your commits](http://davidwalsh.name/squash-commits-git), and it’s good to go.

**Comment by tresni** [comment]
Sorry somehow managed to fuck up the initial rebase... This should take care of it.

**Pull request merged**